web/vibe/web/web.d

```diff
@@ -378,6 +378,12 @@ void redirect(string url)
 	ctx.res.redirect(fullurl);
```

Comment:
I suppose `status` should be passed here....

Reply:
And `found` instead of `Found` ("deprecated")